test/functional/fixtures/es_archiver/README.md

```diff
@@ -0,0 +1,10 @@
+## Changing test data sets
```

Comment:
Is this README complete? or should we add info about OSS server?

Reply:
Running `node scripts/functional_tests_server.js` already starts the server in OSS mode.
So I think the steps listed here are correct.